stetho/src/main/java/com/facebook/stetho/common/android/ViewUtil.java

```diff
@@ -18,47 +20,85 @@
   private ViewUtil() {
   }
 
+  private static boolean isTouchable(View view) {
+    if (view.getVisibility() != View.VISIBLE) {
+      return false;
+    }
+
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.HONEYCOMB) {
+      if (getAlphaHoneycomb(view) < 0.001f) {
+        return false;
+      }
+    }
+
+    return true;
+  }
+
+  @TargetApi(Build.VERSION_CODES.HONEYCOMB)
+  private static float getAlphaHoneycomb(View view) {
+    return view.getAlpha();
+  }
+
   @Nullable
-  public static View hitTest(View view, float x, float y) {
-    return hitTest(view, x, y, null /* viewSelector */);
+  public static View hitTestTouch(View view, float x, float y) {
+    return hitTestTouch(view, x, y, null /* viewSelector */);
   }
 
   // x,y are in view's local coordinate space (relative to its own top/left)
   @Nullable
-  public static View hitTest(View view, float x, float y, @Nullable Predicate<View> viewSelector) {
+  public static View hitTestTouch(
+      View view,
+      float x,
+      float y,
+      @Nullable Predicate<View> viewSelector) {
+    View result = hitTestTouchImpl(view, x, y, viewSelector, false);
+    if (result == null) {
+      result = hitTestTouchImpl(view, x, y, viewSelector, true);
+    }
+    return result;
+  }
+
+  private static View hitTestTouchImpl(
+      View view,
+      float x,
+      float y,
+      @Nullable Predicate<View> viewSelector,
+      boolean allowViewGroupResult) {
+    if (!isTouchable(view)) {
```

Comment:
Non-touchable views should still be selectable using the inspect button.  The purpose is to find views in the tree, not mimic touch behaviour...